test/helper.js

```diff
@@ -0,0 +1,13 @@
+export function transformToStrs(keys, chunkLength = 5) {
```

Comment:
with the inline expectations in the test module above, we should be able to drop this helper.